ocaml/idl/datamodel_errors.ml

```diff
@@ -1092,7 +1094,9 @@ let _ =
   error Api_errors.cluster_force_destroy_failed ["cluster"]
     ~doc:"Force destroy failed on a Cluster_host while force destroying the cluster." ();
   error Api_errors.clustering_enabled_on_network ["network"]
-    ~doc:"The network has cluster objects attached." ()
+  ~doc:"The network has cluster objects attached." ();
+  error Api_errors.cluster_stack_in_use ["cluster stack"]
```

Comment:
We should say `cluster_stack` here, because this seems to be a kind of ID, we don't have space in any of these.